config/store_config.go

```diff
@@ -24,6 +24,7 @@ type StoreConfig struct {
 	StoreBackend  string `yaml:"storebackend"`  // storage backend (e.g. FS, K/V, ...)
 	SyncBackend   string `yaml:"syncbackend"`   // sync backend (e.g. GitCLI, GoGit, ...)
 	UseSymbols    bool   `yaml:"usesymbols"`    // always use symbols when generating passwords
+	Notify        bool   `yaml:"notify"`        // enable desktop notifications
```

Comment:
Maybe it's just me but in a config option to just call it Notify (like the internal library) could be confusing. Maybe EnableNotifications, or just Notifications would be clearer to the outside?

Reply:
Agree! Let's go with Notifications, as it's a bit more concise.